docs/api_output.rst

```diff
@@ -35,7 +35,7 @@ RGBLED
 ======
 
 .. autoclass:: RGBLED(red, green, blue, active_high=True)
-    :members: on, off, red, green, blue, value
+    :members: on, off, blink, red, green, blue, value
```

Comment:
Dunno if there's any 'need' for it, but for the sake of completeness would it be worth adding `initial_value` to `RGBLED.__init__` ?